packages/run-protocol/src/psm/psm.js

```diff
@@ -0,0 +1,170 @@
+// @ts-check
+import '@agoric/zoe/exported.js';
+import '@agoric/zoe/src/contracts/exported.js';
+import '@agoric/governance/src/exported';
+import { assertProposalShape } from '@agoric/zoe/src/contractSupport/index.js';
+import {
+  ceilMultiplyBy,
+  makeRatio,
+} from '@agoric/zoe/src/contractSupport/ratio.js';
+import { Far } from '@endo/marshal';
+// import { CONTRACT_ELECTORATE } from '@agoric/governance';
+
+import { AmountMath } from '@agoric/ertp';
+import { makeMakeCollectFeesInvitation } from '../vaultFactory/collectRewardFees.js';
+
+const { details: X } = assert;
+
+const BASIS_POINTS = 10000n;
+
```

Comment:
Please add a top comment explaining the point of the contract. We usually also document the keywords in use and the main invitations. It will be much easier fro someone to add contract docs to our documentation site if we have at least a rudimentary description here.